src/tools/fsync/test_fsync.c

```diff
@@ -69,7 +69,7 @@ main(int argc, char *argv[])
 	for (i = 0; i < XLOG_SEG_SIZE; i++)
 		full_buf[i] = 'a';
 
-	if ((tmpfile = open(filename, O_RDWR | O_CREAT, S_IRUSR | S_IWUSR, 0)) == -1)
```

Comment:
Not sure what the file ,0 would have been for, but the new code is replacement with the code in backend//storage/file/fd.c